godotopenxrlynx/NOTICE.txt

```diff
@@ -1,309 +1,309 @@
-=====================================================================================

```

Comment:
We should add exceptions for the `NOTICE` and `LICENSE` files.

Reply:
Exceptions added in `file_format.sh`